subsys/bluetooth/controller/ll_sw/ull_llcp.h

```diff
@@ -98,6 +98,11 @@ void ull_cp_run(struct ull_cp_conn *conn);
  */
 void ull_cp_rx(struct ull_cp_conn *conn, struct node_rx_pdu *rx);
 
+/**
```

Comment:
A thought that occured to me just now: should we re-order these prototypes in the same order as the control procedures are mentioned in the BT spec.?

Reply:
Fine with me, but it should be done in a separate PR.

Reply:
Agreed